### PR TITLE
0.39.1 release prep

### PR DESCRIPTION
**Pull request by Matthew-Jennings**

### CODE REVIEW

**Review by SimonBiggs — APPROVED** [review]
LGTM :)

File: lib/pymedphys/docs/contrib/info/release-guide.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # pymedphys Release Procedure
 
-.. note::
-Please ensure that you have followed the [setup guide](https://docs.pymedphys.com/en/latest/contrib/setups/index.html)
+
+> **NOTE:** Please ensure that you have followed the [setup guide](https://docs.pymedphys.com/en/latest/contrib/setups/index.html)
```
Comment (SimonBiggs) [inline lib/pymedphys/docs/contrib/info/release-guide.md:4]: If you want you can change this to:

```{note}
Here is a note
```

If you want to use the note directive within markdown. See https://jupyterbook.org/en/stable/content/myst.html#directives

Comment (Matthew-Jennings) [reply lib/pymedphys/docs/contrib/info/release-guide.md:4]: Thanks!

Would you mind if I made the change after the release? The CI is almost done... haha

Comment (SimonBiggs) [reply lib/pymedphys/docs/contrib/info/release-guide.md:4]: Of course, you can also feel free to not change it :)

Comment (SimonBiggs) [reply lib/pymedphys/docs/contrib/info/release-guide.md:4]: Also, you don't need to wait for the LGTM analysis

**Pull request merged**